features/features.go

```diff
@@ -89,6 +89,11 @@ type Config struct {
 	// StoreARIReplacesInOrders causes the SA to store and retrieve the optional
 	// ARI replaces field in the orders table.
 	StoreARIReplacesInOrders bool
+
+	// DNSAccount01Enabled enables or disables support for the dns-account-01
```

Comment:
This comment should indicate which services respect this flag (i.e. which services it needs to be set in to be fully successful -- don't want to enable it on the RA but not the VA and end up in a broken state).